translations/README.cs.md

```diff
@@ -4,14 +4,14 @@
 
 ## Seznam obsahu
 
-- [O projektu](#o%20projektu)
-- [Použité nástroje](#pou%C5%BEit%C3%A9%20n%C3%A1stroje)
+- [O projektu](#o-projektu)
+- [Použité nástroje](#Použité-nástroje)
 - [Instalace](#instalace)
-- [Demo](#%C5%BEiv%C3%A9%20demo)
+- [Demo](#Živé-demo)

```

Comment:
Probably, when you start a link with uppercase, it'll not be more a link. Try to change:

- [ ] `P` to `p`
- [ ] `Ž` to `ž`

Reply:
fixed